src/net/sourceforge/kolmafia/session/ResultProcessor.java

```diff
@@ -1185,7 +1185,7 @@ else if ( resultName.equals( AdventureResult.SUBSTATS ) )
 				long diff = KoLCharacter.calculateBasePoints( stat + count ) -
 					    KoLCharacter.calculateBasePoints( stat );
 				AdventureResult.SESSION_SUBSTATS[ 0 ] += count;
-				AdventureResult.SESSION_FULLSTATS[ 0 ] += diff;
+				AdventureResult.SESSION_FULLSTATS[ 0 ] += (int) diff;
```

Comment:
AdventureResult.SESSION_FULLSTATS[  ] might perhaps be changed to long[] but that has ripple effects